lib/queries/batchRemove.js

```diff
@@ -16,11 +16,22 @@ export default ({ table, returnCols, primaryKey = ['id'] }) => {
 
         const where = whereQuery(combineLiterals(cleanIds), selector);
 
-        const sql = `DELETE FROM ${table} ${where} ${returning};`;
+        const permanentFiltersKeys = Object.keys(permanentFilters);
+        let permanentWhere = '';
+        let permanentParameters = {};
+        if (permanentFiltersKeys.length) {
+            permanentParameters = sanitizeIdentifier(permanentFiltersKeys, permanentFilters);
```

Comment:
no need to sanitize permanentFilter, they do not come from the user

Reply:
Besides if you call this method with an object and its key, it will do absolutely nothing.